src/app/modules/spcp/spcp.types.ts

```diff
@@ -4,6 +4,7 @@ export enum JwtName {
   SP = 'jwtSp',
   CP = 'jwtCp',
   SGID = 'jwtSgid',
+  MyInfo = 'MyInfoCookie',
```

Comment:
I'm not sure this is the right fix. notice that the enum is defined in the SPCP module, because we deliberately want to keep SPCP and MyInfo decoupled (actually SGID shouldn't be here either). we should avoid a situation where we accidentally loosen the types in the SPCP module just for this fix. what do you think about just adding a switch statement in the controller to determine what cookie name to clear?

Reply:
thanks! agree, have cleaned up